src/BlockTypes/MiniCartContents.php

```diff
@@ -122,6 +126,8 @@ protected function enqueue_assets( array $attributes ) {
 		$parsed_style = '';
 
 		foreach ( $styles as $style ) {
+			$selector = is_array( $style['selector'] ) ? implode( ', ', $style['selector'] ) : $style['selector'];
```

Comment:
Nitpicking, but if we want to save some characters, we can remove the space here...
```suggestion
			$selector = is_array( $style['selector'] ) ? implode( ',', $style['selector'] ) : $style['selector'];
```